package.json

```diff
@@ -59,7 +59,9 @@
   "engines": {
     "node": ">=12.14.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jsonpath": "^1.1.1"
```

Comment:
We keep all dependencies as dev dependencies and people who wish to use 'jsonPath' would need to install it separately like they do with 'node-fetch'

Reply:
@mhagmajer fixed, thanks :)